client/driver/rkt_test.go

```diff
@@ -0,0 +1,102 @@
+package driver
+
+import (
+        "fmt"
+        "os"
+        "os/exec"
+        "testing"
+
+        "github.com/hashicorp/nomad/client/config"
+        "github.com/hashicorp/nomad/nomad/structs"
+
+        ctestutils "github.com/hashicorp/nomad/client/testutil"
+)
+
+// rktLocated looks to see whether rkt binaries are available on this system
+// before we try to run tests. We may need to tweak this for cross-OS support
+// but I think this should work on *nix at least.
+func rktLocated() bool {
+        _, err := exec.Command("rkt", "version").CombinedOutput()
+        return err == nil
+}
```

Comment:
I don't see the value in checking if we are on non-Windows and running as root if we aren't actually going to execute Rocket. Let me know if I'm missing something, though.